.github/workflows/pull-request-test.yml

```diff
@@ -62,38 +66,60 @@ jobs:
+      test-legacy-enabled: ${{ steps.deploy-comment.outputs.test-legacy-enabled}}
       extra-values: ${{ steps.deploy-comment.outputs.extra-values}}
     steps:
-      - uses: actions/checkout@v4.1.7
       - id: deploy-comment
-        uses: SwissDataScienceCenter/renku-actions/check-pr-description@v1.14.1
+        uses: SwissDataScienceCenter/renku-actions/check-pr-description@v1.14.2
         with:
-          string: /deploy
           pr_ref: ${{ github.event.number }}
 
   deploy-pr:
     name: Deploy
-    if: github.event.action != 'closed'
-    needs: [check-deploy]
     runs-on: ubuntu-24.04
+    needs: [check-deploy]
+    permissions:
+      pull-requests: write
+    if: github.event.action != 'closed'
     environment:
       name: ci-renku-${{ github.event.number }}
       url: https://ci-renku-${{ github.event.number }}.dev.renku.ch
     steps:
       - uses: actions/checkout@v4.1.7
+      - name: Find deplyoment url
+        if: needs.check-deploy.outputs.pr-contains-string == 'true'
+        uses: peter-evans/find-comment@v3
+        id: deploymentUrlMessage
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: "RenkuBot"
+          body-includes: "You can access the deployment of this PR at"
+      - name: Create comment deployment url
+        if: steps.deploymentUrlMessage.outputs.comment-id == 0 &&
+          needs.check-deploy.outputs.pr-contains-string == 'true'
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          token: ${{ secrets.RENKUBOT_GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            You can access the deployment of this PR at https://ci-renku-${{ github.event.number }}.dev.renku.ch
       - name: renku build and deploy
         if: needs.check-deploy.outputs.pr-contains-string == 'true'
-        uses: SwissDataScienceCenter/renku-actions/deploy-renku@v1.14.1
+        uses: SwissDataScienceCenter/renku-actions/deploy-renku@v1.14.2
         env:
           DOCKER_PASSWORD: ${{ secrets.RENKU_DOCKER_PASSWORD }}
           DOCKER_USERNAME: ${{ secrets.RENKU_DOCKER_USERNAME }}
           GITLAB_TOKEN: ${{ secrets.DEV_GITLAB_TOKEN }}
           KUBECONFIG: "${{ github.workspace }}/renkubot-kube.config"
-          RENKU_ANONYMOUS_SESSIONS: true
+          RANCHER_DEV_API_ENDPOINT: ${{ secrets.RANCHER_DEV_API_ENDPOINT }}
+          RANCHER_PROJECT_ID: ${{ secrets.CI_RANCHER_PROJECT }}
+          RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
           RENKU_RELEASE: ci-renku-${{ github.event.number }}
+          RENKU_TESTS_ENABLED: true
           RENKU_VALUES_FILE: "${{ github.workspace }}/values.yaml"
           RENKU_VALUES: ${{ secrets.COMBINED_CHARTS_CI_RENKU_VALUES }}
           RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_DEV_KUBECONFIG }}
+          RENKUBOT_RANCHER_BEARER_TOKEN: ${{ secrets.RENKUBOT_RANCHER_BEARER_TOKEN }}
```

Comment:
issue: as dicsussed offline the extra env variables are not needed.

Reply:
Fixed in https://github.com/SwissDataScienceCenter/renku/pull/3948/commits/1564b71abadd24b3ce2f6f9838c1f3d9548eb75c